packages/reporter/src/commands/command.tsx

```diff
@@ -40,8 +40,9 @@ export const formattedMessage = (message: string) => {
   const spaceEscapedText = textToConvert.replace(/^ +/gm, (initialSpaces) => '&#32;'.repeat(initialSpaces.length)) // &#32 is the HTML entity for a space
   const converted = md.renderInline(spaceEscapedText)
   const assertion = (split[1] && [`<strong>${split[1].trim()}</strong>`]) || []
+  const withLineBreaks = converted.replace(/\n/g, '<br>')
```

Comment:
It looks like `markdown-it` has built-in support for this through the [`breaks` parameter](https://markdown-it.github.io/markdown-it/#MarkdownIt.new) (which defaults to false). We should be able to update line 27 to something like this to get the same behavior

```
const md = new Markdown({ breaks: true })
```